support-frontend/app/controllers/CreateSubscription.scala

```diff
@@ -48,7 +48,7 @@ class CreateSubscription(
     implicit request: AuthRequest[CreateSupportWorkersRequest],
     validator: CreateSupportWorkersRequest => Boolean
   ): Future[Result] = {
-    SafeLogger.info(s"[${request.uuid}] User ${request.user.id} is attempting to create a new ${request.body.product} subscription")
+    SafeLogger.info(s"[${request.uuid}] User ${request.user.user.id} is attempting to create a new ${request.body.product} subscription")
```

Comment:
A nit pick that would probably be more work than it's worth to do anything with:
I find these ambiguous situations a bit confusing, they are both users, but is there a way of showing their distinguishing features in the name?
Not sure if it applies in this case, but a technique I use in value classes sometimes is to call the underlying value `value` and the actual class represents the actual concept
`case class User(value: String)`
rather than
`case class User(user: String)`

It does mean you end up with loads of `something.value` which ends up losing a bit of type safety, so there would be other solutions like `user.userString`

Reply:
The reason why we now have `user.user` littered over the code base is because I've stopped using the [implicit def](https://github.com/guardian/identity-auth/blob/cb625a5f28ca3a07512a1cf721a0e1c2d95b5314/src/main/scala/com/gu/identity/play/AuthenticatedIdUser.scala#L28) which converts a `AuthenticatedUser` to an `IdMinimalUser`. I could do a rename e.g. instances of `AuthenticatedUser` could be renamed to `authenticatedUser` (?)

Reply:
right got it, getting rid of implicit conversions is better than not having clutter so it's surely the right decision you've made!
In that case yes either rename as you suggest and/or rename the second `.user` to be `minimalUser` whatever makes more sense.

Reply:
Ok, renaming to `minimalUser` as a stop-gap. Hopefully in the future, these case classes can get edited.